### ISSUE TITLE
Build header component

### ISSUE THREAD

**Issue by comrade-kirby** [HiraethOctoberStrike/octoberstrike.com#14]
Blocked by #16 

Create a reusable component for the header. Logo will be added in a separate issue when it's available.

![image](https://user-images.githubusercontent.com/19193721/127042216-fcb0627e-c5db-43bb-9cb3-5686b79bdac4.png)

[Figma](https://www.figma.com/file/zBXRu0Gb5WquL5FXddFG1u/strike?node-id=0%3A1)

**tips**
- look to button.svelte for reference in how to make and implement a component
- button component can also be referenced for linking between pages
